src/js/components/CollapsibleErrorMessage.js

```diff
@@ -108,13 +108,9 @@ class CollapsibleErrorMessage extends React.Component {
       return null;
     }
 
-    // Make sure at least one whitespace character
-    // exists after the message string
-    message += ' ';
-
     // Render the fixed part of the message
     return (
-        <div key="fixed" className="collapsible-fixed text-align-center text-danger flush-top">
+        <div className="collapsible-fixed text-align-center text-danger flush-top">
```

Comment:
It is, but the key is not needed any more, since I am not using the array approach.
The `floating` is now called `toggled`